Update  Personal Access Token and API URL setup instructions

sdopoku opened the pull request:
This PR proposes two changes:

1. Personal Access Token Steps: Update the legacy API access token instructions for using the Personal Access Token (PAT) in dbt Cloud. 
2.  DBT_CLOUD_BASE_URL: Update instructions in the README and value in create_project.py to find and replace the Access URL for the sandbox account being set up.

sdopoku committed:
Update README.md

With personal access tokens deprecated in dbt Cloud, the setup.py script will need to use the script runner's personal access token (PAT).

sdopoku committed:
Update BASE_URL Steps

Add instructions to find and update the DBT_CLOUD_BASE_URL in the create_project.py file.

bethanyhipple-dbtlabs commented on README.md:
```diff
@@ -103,6 +104,13 @@ Create a new dbt Cloud account and name it with the format "CLIENT Group dbt Lea
 
 3. Create a new file using the private template in the [config](config) folder. Name it based on the date of the training and the client name. Update the fields for the upcoming training.
 
+4. Open the `create_project.py` in the `config/dbt` folder and update the DBT_CLOUD_BASE_URL. To update the correct value, find the Access URL by going Account settings > Account > Access URL.  E.g. If your Access URL is `https://a1.bc2.dbt.com`, your DBT_CLOUD_BASE_URL will be `https://a1.bc2.dbt.com/api/v3/`. Pay attention to include the `api/v3` at the end of the URL.
```
tweak -- change number ordering here. It looks like you have two 4s

bethanyhipple-dbtlabs commented on README.md:
````diff
@@ -46,7 +46,7 @@ A repo for documentation and code involved in setting up different dbt Learn exp
   ```
 
-  - The dbt_cloud_token is accessible from dbt Cloud under Your Profile > Personal profile > API
+  - The dbt_cloud_token is accessible by Creating a new Personal access token (PAT) from dbt Cloud under Your Profile > API tokens > Personal tokens. Pay attention to create a PAT and not a Service token. Your PAT should start with a dbtu_ prefix.
````
suggestion: add a piece about creating this PAT in the sandbox account for the GT since PATs will change account to account.

sdopoku committed:
Correct number bullet ordering

Fix the number ordering to remove two 4's.

sdopoku committed:
Add instructions about creating PAT

Personal Access Tokens (PAT) need to be created for each group training sandbox account.

I moved the instructions under the `Set Up a New Learn` section to clarify this needs to be done for each group training setup.

sdopoku commented:
@bethanyhipple-dbtlabs I've implemented the change and suggestion. Let me know if there's anything else.
